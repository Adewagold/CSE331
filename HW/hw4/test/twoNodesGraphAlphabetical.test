# Same as twoNodeGraph.test, but reverse the 
# order of node adding to test if it prints 
# out in alphabetical order

# Create a graph
CreateGraph two_nodes_graph_alphabetical

# Add two nodes in reverse alphabetical order
AddNode two_nodes_graph_alphabetical b
AddNode two_nodes_graph_alphabetical a

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_graph_alphabetical
ListChildren two_nodes_graph_alphabetical a
ListChildren two_nodes_graph_alphabetical b