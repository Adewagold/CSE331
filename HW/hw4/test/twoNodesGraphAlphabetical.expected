# Same as twoNodeGraph.test, but reverse the 
# order of node adding to test if it prints 
# out in alphabetical order

# Create a graph
created graph two_nodes_graph_alphabetical

# Add two nodes in reverse alphabetical order
added node b to two_nodes_graph_alphabetical
added node a to two_nodes_graph_alphabetical

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_graph_alphabetical contains: a b
the children of a in two_nodes_graph_alphabetical are:
the children of b in two_nodes_graph_alphabetical are: